python/orca/src/bigdl/orca/learn/pytorch/torch_runner.py

```diff
@@ -131,6 +131,7 @@ def __init__(self,
         self.sync_stats = sync_stats
         self.epoch_stats = None  # The state saved in every epoch
         self._mode = 'val'  # By default we don't use ddp model
+        self.pocket = dict() # Used to store some customized attributes
```

Comment:
What's the difference between using a dict VS directly assigning `runner.images = ...` in the hook?

Reply:
> What's the difference between using a dict VS directly assigning `runner.images = ...` in the hook?

To avoid users assign and occupy runner's reserved attrs like `batch` or `loss` by mistake, so we open up a new field for them to store whatever they want.

Reply:
Theoretically if users don't assign the reserved values, the two ways are equivalent right?
So the question is do we have way to prevent users to override reserved values? Probably no for now?

Reply:
> Theoretically if users don't assign the reserved values, the two ways are equivalent right? So the question is do we have way to prevent users to override reserved values? Probably no for now?

Yes, but we can keep this for now i think? For 1. most of time users are not awared of what attribute names are availble.  2. There two kinds of way of assignment are not conflicting.